src/Provider/Facebook.php

```diff
@@ -87,6 +94,25 @@ public function __construct($options = [], array $collaborators = [])
         if (!empty($options['enableBetaTier']) && $options['enableBetaTier'] === true) {
             $this->enableBetaMode = true;
         }
+
+        if (!empty($options['fields']) && is_array($options['fields'])) {
+            array_walk($options['fields'], static function ($value) { if (!is_string($value)) {
+                throw new FacebookProviderException('Facebook does not support non-string fields');
+            }});
```

Comment:
I don't think this is necessary. Configuration should be the same dev/prod, and someone will realize pretty quickly if the fields are not valid.

Reply:
Fair enough, I wasn't sure which level of validation was desired. PR updated.